openmetadata-ui/src/main/resources/ui/src/components/common/entityPageInfo/ManageButton/ManageButton.tsx

```diff
@@ -244,6 +313,18 @@ const ManageButton: FC<Props> = ({
           onCancel={() => setIsDelete(false)}
         />
       )}
+      {onEditDisplayName && (
+        <EntityNameModal
+          disableNameUpdate
+          entity={{
+            name: entityName,
+            displayName,
+          }}
+          visible={isDisplayNameEditing}
+          onCancel={() => setIsDisplayNameEditing(false)}
+          onSave={handleDisplayNameUpdate}
+        />
+      )}
```

Comment:
It is being visible based on other flag, not sure if we should load it conditionally.
Any issue in rendering it normally?

Reply:
onEditDisplayName is an optional field, so if its present then only we need to show the field.